suite/tests/api/dis-a64.txt

```diff
@@ -13562,69 +13562,69 @@ d5087a5c : dc csw, x28                    : dc_csw %x28
-d50b7a33 : dc cvac, x19                   : dc_cvac (%x19)[1byte]
-d50b7a34 : dc cvac, x20                   : dc_cvac (%x20)[1byte]
-d50b7a35 : dc cvac, x21                   : dc_cvac (%x21)[1byte]
-d50b7a36 : dc cvac, x22                   : dc_cvac (%x22)[1byte]
-d50b7a37 : dc cvac, x23                   : dc_cvac (%x23)[1byte]
-d50b7a38 : dc cvac, x24                   : dc_cvac (%x24)[1byte]
-d50b7a39 : dc cvac, x25                   : dc_cvac (%x25)[1byte]
-d50b7a3a : dc cvac, x26                   : dc_cvac (%x26)[1byte]
-d50b7a3b : dc cvac, x27                   : dc_cvac (%x27)[1byte]
-d50b7a3c : dc cvac, x28                   : dc_cvac (%x28)[1byte]
-d50b7a3d : dc cvac, x29                   : dc_cvac (%x29)[1byte]
-d50b7a3e : dc cvac, x30                   : dc_cvac (%x30)[1byte]
-
-d50b7b20 : dc cvau, x0                    : dc_cvau (%x0)[1byte]
-d50b7b21 : dc cvau, x1                    : dc_cvau (%x1)[1byte]
-d50b7b22 : dc cvau, x2                    : dc_cvau (%x2)[1byte]
-d50b7b23 : dc cvau, x3                    : dc_cvau (%x3)[1byte]
-d50b7b24 : dc cvau, x4                    : dc_cvau (%x4)[1byte]
-d50b7b25 : dc cvau, x5                    : dc_cvau (%x5)[1byte]
-d50b7b26 : dc cvau, x6                    : dc_cvau (%x6)[1byte]
-d50b7b27 : dc cvau, x7                    : dc_cvau (%x7)[1byte]
-d50b7b28 : dc cvau, x8                    : dc_cvau (%x8)[1byte]
-d50b7b29 : dc cvau, x9                    : dc_cvau (%x9)[1byte]
-d50b7b2a : dc cvau, x10                   : dc_cvau (%x10)[1byte]
-d50b7b2b : dc cvau, x11                   : dc_cvau (%x11)[1byte]
-d50b7b2c : dc cvau, x12                   : dc_cvau (%x12)[1byte]
-d50b7b2d : dc cvau, x13                   : dc_cvau (%x13)[1byte]
-d50b7b2e : dc cvau, x14                   : dc_cvau (%x14)[1byte]
-d50b7b2f : dc cvau, x15                   : dc_cvau (%x15)[1byte]
-d50b7b30 : dc cvau, x16                   : dc_cvau (%x16)[1byte]
-d50b7b31 : dc cvau, x17                   : dc_cvau (%x17)[1byte]
-d50b7b32 : dc cvau, x18                   : dc_cvau (%x18)[1byte]
-d50b7b33 : dc cvau, x19                   : dc_cvau (%x19)[1byte]
-d50b7b34 : dc cvau, x20                   : dc_cvau (%x20)[1byte]
-d50b7b35 : dc cvau, x21                   : dc_cvau (%x21)[1byte]
-d50b7b36 : dc cvau, x22                   : dc_cvau (%x22)[1byte]
-d50b7b37 : dc cvau, x23                   : dc_cvau (%x23)[1byte]
-d50b7b38 : dc cvau, x24                   : dc_cvau (%x24)[1byte]
-d50b7b39 : dc cvau, x25                   : dc_cvau (%x25)[1byte]
-d50b7b3a : dc cvau, x26                   : dc_cvau (%x26)[1byte]
-d50b7b3b : dc cvau, x27                   : dc_cvau (%x27)[1byte]
-d50b7b3c : dc cvau, x28                   : dc_cvau (%x28)[1byte]
-d50b7b3d : dc cvau, x29                   : dc_cvau (%x29)[1byte]
-d50b7b3e : dc cvau, x30                   : dc_cvau (%x30)[1byte]
+d50b7a20 : dc cvac, x0                    : dc_cvac (%x0)[64byte]
```

Comment:
Does this mean that we're assuming the cache line size to be 64 bytes in this test? Will it fail if run on a system with a different line size? Maybe add a comment.

Reply:
> Does this mean that we're assuming the cache line size to be 64 bytes in this test? Will it fail if run on a system with a different line size? 

Yes. I don't think any changes are required in the disassembler, but we do need to modify the harness for codec testing. I'll add a comment if the change doesn't need to be done for v9.0.